sktime/utils/plotting.py

```diff
@@ -30,6 +30,22 @@ def plot_series(
 ):
     """Plot one or more time series.
 
+    Description
```

Comment:
Thanks!

can you kindly remove the "description" header, this is not an official numpydoc header.